app/components/npq_separation/admin/outcomes_table_component.rb

```diff
@@ -0,0 +1,43 @@
+module NpqSeparation
+  module Admin
+    class OutcomesTableComponent < ViewComponent::Base
+      attr_reader :outcomes
+
+      def initialize(outcomes)
+        @outcomes = outcomes
+      end
+
+      def call
+        if outcomes.none?
+          tag.p "No outcomes recorded.", class: "govuk-body"
+        else
+          render(GovukComponent::TableComponent.new(head:, rows:))
+        end
+      end
+
+    private
+
+      def head
+        [
+          "Outcome",
+          "Completion date",
+          "Submitted by provider",
+          "Sent to TRA API",
+          "Recorded by TRA API",
+        ]
+      end
+
+      def rows
+        outcomes.sort_by(&:created_at).reverse.map do |outcome|
+          [
+            outcome.state.humanize,
+            outcome.completion_date.to_fs(:govuk_short),
+            outcome.created_at.to_date.to_fs(:govuk),
+            outcome.sent_to_qualified_teachers_api_at.try(:to_fs, :govuk_short).presence || "No",
+            helpers.boolean_red_green_nil_tag(outcome.qualified_teachers_api_request_successful),
```

Comment:
I think we have some missing logic vs what is in ECF regarding when we should show this value, let's catchup tomorrow on this. Also worth mentioning that usually outcomes are associated with a declaration so maybe we should show them after their respective declaration? Anyway let's chat tomorrow :)